lib/game_of_life_web/live/universe_live.ex

```diff
@@ -25,9 +27,20 @@ defmodule GameOfLifeWeb.UniverseLive do
     {:ok, put_generation(socket, &Universe.info/1)}
   end
 
-  def handle_info(:tick, socket), do: {:noreply, put_generation(socket, &Universe.tick/1)}
+  def handle_info(:tick, socket) do 
+    schedule_tick(socket)
+    {:noreply, put_generation(socket, &Universe.tick/1)}
```

Comment:
I think the `put_generation` should be called before scheduling the next job.  Otherwise a really fast `speed` could result in a bunch of queued ticks.